walkeeper/src/safekeeper.rs

```diff
@@ -282,6 +282,8 @@ pub struct AppendResponse {
     pub commit_lsn: Lsn,
     // Min disk consistent lsn of pageservers (portion of WAL applied and written to the disk by pageservers)
     pub disk_consistent_lsn: Lsn,
+    // instance size of the timeline
+    pub instance_size: u64,
```

Comment:
That breaks protocol =/ @arssher @petuhovskiy may be switch hs_feedback to some binary json format so we can add new fields easily?

Reply:
Also `instance_size` sounds like a static limit value. May be `current_size` (we also have a total one which includes sliced WAL) is better?

Reply:
It is already a custom message, isn't it?
AppendResponse on postgres side is handled by `HandleWalKeeperResponse()`